sdk/trace/tracer.go

```diff
@@ -17,10 +25,35 @@ type tracer struct {
 
 	provider             *TracerProvider
 	instrumentationScope instrumentation.Scope
+
+	selfObservabilityEnabled bool
+	spanLiveMetric           otelconv.SDKSpanLive
+	spanStartedMetric        otelconv.SDKSpanStarted
 }
 
 var _ trace.Tracer = &tracer{}
 
+func (tr *tracer) initSelfObservability() {
+	env := os.Getenv("OTEL_GO_X_SELF_OBSERVABILITY")
+	if strings.ToLower(env) != "true" {
+		return
+	}
```

Comment:
What is the reason not to use `x.SelfObservability` here?

Reply:
I have forgotten to do that 😅

Reply:
ad13ce9b